libs/ui-react/src/lib/Components/Link/Link.stories.tsx

```diff
@@ -0,0 +1,317 @@
+    onClick,
+  },
+  parameters: {
+    docs: {
+      source: {
+        code: `
+<Link
+  appearance="underlined"
+  icon={Information}
+  href="#"
+>
+  Link with Icon
+</Link>
+`,
+      },
+    },
+  },
+};
+
+export const External: Story = {
+  args: {
+    appearance: 'underlined',
+    children: 'External Link',
+    isExternal: true,
+    href: 'https://example.com',
+    onClick,
+  },
+  parameters: {
+    docs: {
+      source: {
+        code: `
+<Link
+  appearance="underlined"
+  isExternal
+  href="https://example.com"
+>
+  External Link
+</Link>
+`,
+      },
+    },
+  },
+};
+
+export const ExternalLinkWithIcon: Story = {
+  args: {
+    appearance: 'underlined',
+    children: 'External Link with Icon',
+    icon: Information,
+    isExternal: true,
+    href: 'https://example.com',
+    onClick,
+  },
+  parameters: {
+    docs: {
+      source: {
+        code: `
+  <Link
+    appearance="underlined"
+    icon={Information}
+    isExternal
+    href="https://example.com"
+  >
+    External Link with Icon
+  </Link>
+  `,
+      },
+    },
+  },
+};
+
+export const AppearanceShowcase: Story = {
+  render: () => {
+    const appearances: Array<{ name: string; appearance: LinkAppearance }> = [
+      { name: 'Underlined', appearance: 'underlined' },
+      { name: 'Accent', appearance: 'accent' },
+    ];
+
+    return (
+      <div className="flex gap-32 p-8">
+        {appearances.map(({ name, appearance }) => (
+          <Link
+            key={appearance}
+            appearance={appearance}
+            href="#"
+            onClick={onClick}
+            icon={Information}
+            isExternal
+          >
+            {name}
+          </Link>
+        ))}
+      </div>
+    );
+  },
+};
+
+export const SizesShowcase: Story = {
+  render: () => (
+    <>
+      <div className="flex items-center gap-32 p-8">
+        <Link
+          appearance="underlined"
+          size="sm"
+          href="#"
+          onClick={onClick}
+          icon={Information}
+          isExternal
+        >
+          Small
+        </Link>
+        <Link
+          appearance="underlined"
+          size="md"
+          href="#"
+          onClick={onClick}
+          icon={Information}
+          isExternal
+        >
+          Medium
+        </Link>
+      </div>
+      <div className="flex items-center gap-32 p-8">
+        <Link
+          appearance="accent"
+          size="sm"
+          href="#"
+          onClick={onClick}
+          icon={Information}
+          isExternal
+        >
+          Small Accent
+        </Link>
+        <Link
+          appearance="accent"
+          size="md"
+          href="#"
+          onClick={onClick}
+          icon={Information}
+          isExternal
+        >
+          Medium Accent
+        </Link>
+      </div>
+    </>
+  ),
+};
+
+export const ResponsiveLayout: Story = {
+  render: () => (
+    <div className="flex w-256 flex-col gap-8 rounded-sm bg-canvas p-8">
```

Comment:
can you please change the bg color of the container , it's white on white
```suggestion
    <div className="flex w-256 flex-col gap-8 rounded-sm bg-muted p-8">
```
<img width="471" height="187" alt="image" src="https://github.com/user-attachments/assets/2f1af86e-e673-410f-83b4-6a0eb6e9e62b" />